scripts/data-fetch-old.js

```diff
@@ -0,0 +1,19 @@
+var fs = require('fs')
```

Comment:
We can drop this as it's just or if we want to keep it as an alternative, we could rename it something descriptive (like direct metro open data portal fetch or something).

Reply:
I think you are right - make the name more descriptive.  I didn't want to loose the old code, just in case we run into any issues with the new code and need to revert at some point.